doc/charm++/manual.rst

```diff
@@ -5255,6 +5255,28 @@ across an entire chare array or group or nodegroup. It is yet to be supported fo
 section broadcasts. Additionally, there is also no support for migration of chares that have
 pending RDMA transfer requests.
 
+
+Using Zerocopy Post API in iterative applications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In iterative applications, since different arrays are sent for each iteration, it is important
```

Comment:
The language here is confusing when talking about different buffers. Are you talking about an chare array elements each owning multiple buffers that are sent/recv'ed per iteration, or about posting different buffers for each iteration?

Reply:
I am talking about posting different buffers across iterations, i.e. posting a new buffer for each iteration in order to avoid overwrite on the same buffer.

Reply:
I modified that sentence to something simpler.